metis-data-checker/metis-data-checker-service/src/main/java/eu/europeana/metis/data/checker/service/persistence/RecordDao.java

```diff
@@ -41,13 +41,13 @@
   /**
    * Constructor with required fields.
    *
-   * @param connectionProvider
    * @throws IndexingException In case of problems setting up the indexer.
    */
   @Autowired
   public RecordDao(AbstractConnectionProvider connectionProvider) throws IndexingException {
```

Comment:
Sonarlint: "document the parameter(s): connectionProvider"

Reply:
Done